projects/plugins/jetpack/extensions/blocks/like/tools/update-editor-styles.js

```diff
@@ -0,0 +1,78 @@
+const fs = require( 'fs' );
+const path = require( 'path' );
+
+// URL to fetch CSS from
+const url = 'https://widgets.wp.com/likes/style.css';
+
+// Function to pretty print CSS
+function prettyPrintCSS( css ) {
+	// Adding a line break after each semicolon and opening brace
+	css = css.replace( /;/g, ';\n  ' ).replace( /\{/g, ' {\n  ' );
```

Comment:
Can we replace the spaces `  ` with `\t` to keep the tabs?

Reply:
Fixed in: https://github.com/Automattic/jetpack/pull/35027